risc0/zkvm/platform/src/memory.rs

```diff
@@ -54,12 +54,10 @@ impl Region {
 }
 
 // These should match the linker script in `risc0/build/risc0.ld`.
-pub const INPUT: Region = Region::new(0x0000_0400, mb(32) - kb(1));
```

Comment:
We should probably redistribute the excess region to other sections, like data/heap/text.